setup/Microsoft.VisualStudio.NuGet.BuildTools.vsmanproj

```diff
@@ -10,6 +10,7 @@
     <BuildNumber>$(SemanticVersion).$(BuildNumber)</BuildNumber>
     <TargetName>$(MSBuildProjectName)</TargetName>
     <OutputPath>$(VsixPublishDestination)</OutputPath>
+    <TargetFrameworkVersion>v4.8</TargetFrameworkVersion>
```

Comment:
What was the previous version here? 
Asking because it's only an addition

Reply:
There is no specific version there and it's net4.0.
So it's not able to be built on VS2022, as VS2022 doesn't support net45(or earlier full framework) anymore.
Please see [Visual Studio 2022 Support for .NET Development](https://docs.microsoft.com/en-us/visualstudio/releases/2022/compatibility#-visual-studio-2022-support-for-net-development) for more details.

Reply:
Can we do net472 instead as that's what our projects actually target?

Reply:
Sure, I'll change it into net472. Thanks!